tests/core/gui/test_password_change.py

```diff
@@ -149,3 +149,24 @@ def _create_change_password_dialog():
 
     async with aqtbot.wait_signals([lw.login_with_password_clicked, tabw.logged_in]):
         await aqtbot.mouse_click(lw.button_login, QtCore.Qt.LeftButton)
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+async def test_change_password_canceled(
```

Comment:
Why name it cancel? It just checks the dialog is open?

Reply:
Because it uses `autoclose_dialog` fixture that close the dialog window, so it test that the widget is display and also that nothing happen if it's simply  closed

Reply:
Right, missed that one